src/icons/SealEye.tsx

```diff
@@ -1,9 +1,63 @@
-import { stroke } from 'classnames/tailwind'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
 
-const strokeSecondaryDimmed = stroke('stroke-secondary-dimmed')
-const strokeTertiaryLine = stroke('stroke-tertiary')
+enum SealEyeColor {
+  accent = 'stroke-accent',
+  secondary = 'stroke-secondary-dimmed',
+  tertiary = 'stroke-tertiary',
+}
+
+const commonStrokeWidth = strokeWidth('stroke-1.5')
+const strokeClass = (color: SealEyeColor) =>
+  classnames(commonStrokeWidth, stroke(color))
+
+export default function ({ reversed }: { reversed?: boolean }) {
```

Comment:
Reversed is `colsedEye`?

Reply:
Reversed means `inverted` icon, because it rotated upside down

Reply:
Maybe let's call it `upsideDown`?

Reply:
And if it's rotated, maybe we use `rotate-180`?

Reply:
No, we don't use `rotate` here because it's different icon.

Sure, let's call it `upsideDown`